cloudigrade/account/models.py

```diff
@@ -0,0 +1,15 @@
+"""Cloudigrade Account Models."""
+from django.db import models
+
+from util.models import BaseModel
+
+
+class Account(BaseModel):
+    """Account model."""
+
+    account_id = models.DecimalField(
+        max_digits=12,
+        decimal_places=0,
+        db_index=True
```

Comment:
This table may be small enough that this index won't be used.

Reply:
but Soon™?

Reply:
Yeah, maybe. Even with a few thousand entries I wouldn't be shocked to see the query planner ignore the index as we bang our heads trying to get it to use the index.